packages/demand-bloc-payment/src/PaymentBloc.test.ts

```diff
@@ -140,6 +140,7 @@ describe('PaymentBloc', () => {
       expect(cardsInfoMock.setPaymentCards).toBeCalledWith(cards, payer)
     })
 
+    // @ts-ignore
```

Comment:
What is it used for?

Reply:
it has problem with `done` parameter type

Reply:
ok, we can leave it or use another approach:
https://jestjs.io/docs/tutorial-async#error-handling